.github/workflows/build-php.yaml

```diff
@@ -19,8 +19,12 @@ jobs:
         uses: actions/checkout@v3
       - name: Build PHP
         run: make php/php-${{ matrix.version }}
+      - name: Rename artifacts
+        shell: bash
+        run: |
+          sudo mv php/build-output/php/php-${{ matrix.version }}/bin/php-cgi{,-${{ matrix.version }}.wasm}
```

Comment:
Is this `mv` command correct? I've never used `mv` with a single parameter, so most likely it's performing the right transformation but I don't know it. I wanted to mention it just in case there's a missing space around.

Reply:
Yup, it's a trick on command parameter expansion: if you have "file1.txt" and want to rename it to "file2.txt", you can do `mv file{1,2}.txt`